test/cpp/tensorexpr/test_expr.cpp

```diff
@@ -69,9 +69,9 @@ void testExprLetStmtTest01() {
   Buffer a_buf("a", kFloat, {1});
   Buffer b_buf("b", kFloat, {1});
 
-  ExprHandle load_a = Load::make(a_buf, 0, 1);
+  ExprHandle load_a = Load::make(a_buf, {0}, 1);
```

Comment:
Minor: since this was very common, it might be easier to add an overloaded version that takes a single Index, instead of vector of indices.

Reply:
Here I prefer being explicit (it might also help people to "relearn" how `Load` now works) - it was one of the most frequent changes in this PR, as I had to change all the existing code, but I don't expect it to be that much of annoyance going forward in incremental changes.